bindings/matrix-sdk-ffi/src/room_info.rs

```diff
@@ -63,7 +63,7 @@ impl RoomInfo {
             joined_members_count: room.joined_members_count(),
             highlight_count: unread_notification_counts.highlight_count,
             notification_count: unread_notification_counts.notification_count,
-            notification_mode: room.notification_mode().await.map(Into::into),
+            notification_mode: room.user_defined_notification_mode().await.map(Into::into),
```

Comment:
Don't we need `notification_mode` anymore? Would it make sense to name the `notification_mode` field the same way as the method we're calling, i.e. `user_defined_notification_mode`?

Reply:
Oh yes. Ok, I've renamed it.